internal/repositories/account.go

```diff
@@ -0,0 +1,15 @@
+package repositories
+
+import (
+	"context"
+	"crud-golang/internal/models"
+)
+
+type Repository interface {
```

Comment:
This is actually Account. Repository name does not provide any context around why would you use this repository.

I would name package repository, and you can create one by doing: repository.NewAccount() that will return Account interface